monitoring/scripts/track-cid-backfill/src/queries.ts

```diff
@@ -0,0 +1,353 @@
+import { QueryTypes } from "sequelize";
+import { sequelizeConn } from "./db";
+
+import fs from 'fs-extra'
+
+export const prodContentNodes = [
```

Comment:
just curious, how'd you generate this?

Reply:
Just went to audius.co and threw `await audiusLibs.ServiceProvider.listCreatorNodes()` into the console

Reply:
perf